ts/screens/wallet/PaymentHistoryDetailsScreen.tsx

```diff
@@ -109,7 +112,21 @@ class PaymentHistoryDetailsScreen extends React.Component<Props> {
       color: paymentInfo.color,
       description: paymentInfo.text11
     };
-    const errorDetail = fromNullable(getErrorDescription(payment.failure));
+    const errorDetail = fromNullable(getErrorDescription(payment.failure)).alt(
```

Comment:
Can you add a comment here?

Reply:
Added within https://github.com/pagopa/io-app/pull/2846/commits/192488f9288d848e8dbdaabae1e0e6caa1500086